code/game/machinery/firealarm.dm

```diff
@@ -176,6 +180,7 @@
 	if(user)
 		log_game("[user] triggered a fire alarm at [COORD(src)]")
 	soundloop.start() //Manually pulled fire alarms will make the sound, rather than the doors.
+	SEND_SIGNAL(src, COMSIG_FIREALARM_USED)
```

Comment:
use two separate signals for this, don't call the same signal in multiple different places.
e.g.
COMSIG_FIREALARM_ON_TRIGGER
COMSIG_FIREALARM_ON_RESET